base/src/main/java/com/smartdevicelink/util/NativeLogTool.java

```diff
@@ -126,15 +124,15 @@ private static boolean log(LogTarget ltarg, String source, String logMsg) {
 						bytesWritten = Log.w(source, chunk);
 						break;
 					case Error:
-						bytesWritten = Log.e(source, chunk);
+						bytesWritten = Log.e(source, chunk, null);
 						break;
 				}
 				if (bytesWritten < chunk.length()) {
-					Log.w(TAG, "Calling Log.e: msg length=" + chunk.length() + ", bytesWritten=" + bytesWritten);

```

Comment:
Is there a reason for changing this from log.w to log.e?

Reply:
Changed back to log.w